src/modules/header/header.jsx

```diff
@@ -0,0 +1,177 @@
+  return [expanded, toggle]
+}
+
+const Menu = ({
+  children,
+  label,
+  id,
+  className,
+  expanded,
+  onExpand,
+  tag: Tag = 'div',
+  ...restProps
+}) => {
+  const titleId = `${id}-title`
+  return (
+    <Tag className={classNames.use(styles.menu).join(className)} {...restProps}>
+      <Button
+        role="button"
+        id={titleId}
+        className={styles.menuTitle}
+        aria-haspopup="true"
+        aria-expanded={expanded}
+        onClick={onExpand}
+      >
+        {label}
+      </Button>
+      <ul
+        className={classNames.use(styles.menuList, expanded && styles.expanded)}
+        aria-labelledby={titleId}
+      >
+        {children}
+      </ul>
+    </Tag>
+  )
+}
+
+const Group = ({
+  children,
+  className,
+  id,
+  label,
+  tag: Tag = 'div',
+  ...restProps
+}) => {
+  const titleId = `${id}-title`
+  return (
+    <Tag
+      className={classNames.use(styles.group).join(className)}
+      {...restProps}
+    >
+      <h3 id={titleId} className={styles.groupTitle}>
+        {label}
+      </h3>
+      <ul className={styles.groupList} aria-labelledby={titleId}>
+        {children}
+      </ul>
+    </Tag>
+  )
+}
+
+const Item = ({ children, href, className, ...passProps }) => (
+  <li className={classNames.use(styles.item).join(className)} {...passProps}>
+    <a className={styles.itemLink} href={href}>
+      {children}
+    </a>
+  </li>
+)
+
+const Header = ({ children, className, id, ...passProps }) => {
+  const navId = `${id}-nav`
+  const [navExpanded, toggleNav] = useToggle(false)
+
+  const [expandedMenu, setExpandedMenu] = useState('')
+  const toggleMenu = useCallback((nextId) =>
+    setExpandedMenu((prevId) => (prevId !== nextId ? nextId : ''))
+  )
+
+  return (
+    <AppBar
+      className={classNames.use(styles.container).join(className)}
+      tag="header"
+      {...passProps}
+    >
+      <AppBar.Item className={styles.logoContainer} href="/" tag="a">
+        <Logo className={styles.logo} />
+      </AppBar.Item>
+
+      <AppBar.Item className={styles.main}>{children}</AppBar.Item>
+
+      <AppBar.Item className={styles.toggleContainer}>
+        <Button
+          type="button"
+          className={styles.toggleButton}
+          aria-controls={navId}
+          aria-expanded={navExpanded}
+          aria-label="Toggle navigation"
+          onClick={toggleNav}
+        >
+          <Icon src="#menu" alt="Menu icon" aria-hidden />
+        </Button>
+      </AppBar.Item>
+
+      <AppBar.Item
+        id={navId}
+        className={classNames.use(styles.nav, navExpanded && styles.expanded)}
+        tag="nav"
+      >
+        <Menu
+          id="services"
+          label="Services"
+          className={styles.servicesMenu}
+          expanded={expandedMenu === 'services'}
+          onExpand={() => toggleMenu('services')}
+        >
+          <Item className={styles.servicesOverviewItem} href="/services">
```

Comment:
If it gets integrated to oacore/next, it will break local development. I find such kind of URLs safe enough to go ahead. I didn't want to develop another complex solution for URL management here and kept it simple at the moment.

Reply:
oh, that's true. What about basePath prop for header?

Reply:
Is it really needed? It may get confusing soon if we add `/data-providers` or `/works` or something related to the oacore/search. I am afraid there is no better way to handle this than it's now.